src/auth.rs

```diff
@@ -45,8 +45,8 @@
 //! # Security:
```

Comment:
The errors-section is missing a propagated error, because the `SecretKey` used in `hazardous` can max be 64 bytes for Blake2b. This limitation needs to be mentioned in the docs.